testing/integration/logs_ingestion_test.go

```diff
@@ -99,6 +99,74 @@ func TestLogIngestionFleetManaged(t *testing.T) {
 	})
 }
 
+func TestDebLogIngestFleetManaged(t *testing.T) {
+	info := define.Require(t, define.Requirements{
+		Group: Deb,
+		Stack: &define.Stack{},
+		OS: []define.OS{
+			{
+				Type:   define.Linux,
+				Distro: "ubuntu",
+			},
+		},
+		Local: false,
+		Sudo:  true,
+	})
```

Comment:
I am not saying do it in this PR, but another way of handling the different between deb/RPM vs `install` subcommand would be to define it in the test requirements. An example could be to change to:

```
	info := define.Require(t, define.Requirements{
		Group: Deb,
		Stack: &define.Stack{},
                 Packaging: define.Deb,
		Local: false,
		Sudo:  true,
	})
```

The testing framework could then look at the `Packaging: define.Deb` and determine automatically that this needs to be on an OS that supports deb's. Each OS can say what it supports in its definition. Then when running the framework with `MATRIX="true"` it could run on Ubuntu, Debian, etc. By default it would just pick Ubuntu. Doing it this way would also allow only the `deb` to be copied to the host. Currently with this change, it is going to copy both the `targz` and `deb` to the host, when both are not needed.